plugins/catalog-backend/migrations/20210302150147_refresh_state.js

```diff
@@ -107,29 +90,24 @@ exports.up = async function up(knex) {
   });
 
   await knex.schema.createTable('refresh_state_references', table => {
-    table.comment(
-      'Holds edges between refresh state rows. Every time when an entity is processed and emits another entity, an edge will be stored to represent that fact. This is used to detect orphans and ultimately deletions.',
-    );
+    const textColumn = isMySQL
+      ? table.string.bind(table)
+      : table.text.bind(table);
```

Comment:
This is the only place that does this. How come you made this choice here? 🤔  These IDs and refs should be equally restricted as other id/ref columns I think

Reply:
In older version of Postgres, switching to "string" causes the ordering of results to be different. I am uncertain as to why this happens.

Reply:
Wow, that's really weird 🤔

Reply:
Yeah, it's why we initially changed the test to just look for it being in the array of results. 🤷🏻 I don't think doing this switch here hurts anything, but it looks odd.

Reply:
Here's the specific error:

```bash
FAIL src/service/DefaultEntitiesCatalog.test.ts (38.73 s)
  ● DefaultEntitiesCatalog › entityAncestry › should return the ancestry with multiple parents, "POSTGRES_9"

    expect(received).toEqual(expected) // deep equality

    Expected: ArrayContaining [{"entity": ObjectContaining {"metadata": {"name": "root"}}, "parentEntityRefs": ["k:default/parent1", "k:default/parent2"]}, {"entity": ObjectContaining {"metadata": {"name": "parent1"}}, "parentEntityRefs": ["k:default/grandparent"]}, {"entity": ObjectContaining {"metadata": {"name": "parent2"}}, "parentEntityRefs": ["k:default/grandparent"]}, {"entity": ObjectContaining {"metadata": {"name": "grandparent"}}, "parentEntityRefs": []}]
    Received: [{"entity": {"apiVersion": "a", "kind": "k", "metadata": {"name": "root"}, "spec": {}}, "parentEntityRefs": ["k:default/parent2", "k:default/parent1"]}, {"entity": {"apiVersion": "a", "kind": "k", "metadata": {"name": "parent1"}, "spec": {}}, "parentEntityRefs": ["k:default/grandparent"]}, {"entity": {"apiVersion": "a", "kind": "k", "metadata": {"name": "grandparent"}, "spec": {}}, "parentEntityRefs": []}, {"entity": {"apiVersion": "a", "kind": "k", "metadata": {"name": "parent2"}, "spec": {}}, "parentEntityRefs": ["k:default/grandparent"]}]

      236 |         expect(result.rootEntityRef).toEqual('k:default/root');
      237 |
    > 238 |         expect(result.items).toEqual(
          |                              ^
      239 |           expect.arrayContaining([
      240 |             {
      241 |               entity: expect.objectContaining({ metadata: { name: 'root' } }),

      at toEqual (service/DefaultEntitiesCatalog.test.ts:238:30)
          at runMicrotasks (<anonymous>)
```